sample/src/main/java/co/nimblehq/sample/AuthenService.kt

```diff
@@ -0,0 +1,13 @@
+package co.nimblehq.sample
+
+import co.nimblehq.sample.entity.UserEntity
+import co.nimblehq.survey.sdk.request.LoginRequest
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface AuthenService {
```

Comment:
- the name could be `AuthService`
- I think we should support authentication layer in the sdk instead

Reply:
@luongvo 
- I agree to change the `AuthenService` to `AuthService`
- For the authentication layer, we will do it in another phase. Please consider this part is an extra one we try to do in this phase.